proposals/0046-i18n-domain-support.md

```diff
@@ -0,0 +1,62 @@
+- Start Date: 2023/12/21
+- Reference Issues:
+- Implementation PR: https://github.com/withastro/astro/pull/9143
+
+
+# Summary
+
+First-class support for domain support in i18n routing.
+
+
+# Background & Motivation
+
+Websites that support internationalisation have different requirements. Among these requirements, there's the need to 
+have localised content under different subdomains or domains. 
+
+# Goals
+
+- Support different domains, all driven by the same Astro project;
+- Mix locales that require a different domain, with locales that don't require a different domain;
+
+# Non-Goals
+
+- Force a redirect to users
```

Comment:
Can you explain this more? I think what you mean is that it's not going to redirect users from the domain to a path prefix, is that right?

Reply:
Yes. Also, if a user lands on a domain different from their browser language, we don't plan to do a redirect.